### ISSUE TITLE
[CRASH] UniGetUI 3.1.1 crashes four seconds after initial screen display

### ISSUE THREAD

**Issue by dschulman-repay** [marticliment/UniGetUI#2583]
### Please confirm these before moving forward.

- [X] I have tried deleting a folder named `UniGetUI` under `%UserProfile%\AppData\Local\UniGetUI`.
- [X] I have tried reinstalling UniGetUI.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).

### Describe your crash

Despite several clean reinstalls of UniGetUI 3.1.1, the start screen loads and then the application crashes immediately after it begins searching for installed packages (about 3-4 seconds later).

I did not have this problem in earlier versions of UniGetUI.

### Logs (if possible)

```text
No error reports, messages, or usable debug logs.
```

### More details

My machine is a 64-bit Windows 10 Enterprise v. 2009 box (Windows 22H2, December 2022 Update v. 19045.4651).

**Comment by marticliment** [comment]
Please see if there is any error on Event Viewer

**Comment by dschulman-repay** [comment]
I checked that, and unfortunately nothing associated with the UniGetUI crash is visible in Event Viewer.

**Comment by martinzk** [comment]
I had the same problem. Just installed UniGetUI using winget for the first time on my machine. After starting it told me I needed an update to UniGetUI but it never started the installer after closing and everytime I tried to open UniGetUI it would crash after a few seconds. Running UniGetUI as admin it started its update and now I can run it without admin again.

**Comment by dschulman-repay** [comment]
After UniGetUI.exe crashes, `%errorlevel%` returns `0`.

**Comment by dschulman-repay** [comment]
Here's something really strange: though `UniGetUI.exe` always crashes, `WingetUI.exe` runs - and the two binaries' contents are bit-for-bit identical!

I now suspect that `UniGetUI.exe` might be blocked by my organization's firewall or security software.

**Comment by marticliment** [comment]
Yes, this is really weird. Both executables should have the same content, but what happens if you delete UniGetUI.exe and duplicate WingetUI.exe, renaming the copy to UniGetUI.exe?

**Comment by bdemircan** [comment]
same problem with windows11 pro 10.0.22621

**Comment by KaranocaVe** [comment]
In fact, WingetUI still crashes frequently for me.

**Comment by marticliment** [comment]
Any errors on event viewer?

**Comment by jurcyk** [comment]
Crowdstrike detects this as "Defense Evasion via Reflective Code Loading" and closes it shortly after opening

**Comment by marticliment** [comment]
Then you should report a false positive to CrowdStrike, since UniGetUI is not a virus. The "reflective code loading" is likely caused by the WinGet COM integrations or by the PowerShell integrations (which dynamically load .NET DLLs sometimes)

**Comment by marticliment** [comment]
I am closing this issue as I can't fix it myself

**Comment by GeoSimos** [comment]
Running via the wingetui.exe does work for me as well.
